LevelUp.Pos.ProposedOrders/ProposedOrderCalculator.cs

```diff
@@ -88,23 +94,45 @@ int customerPaymentAmount
             int appliedDiscountAmount
         )
         {
-            AdjustedCheckValues values = CalculateCreateProposedOrderValues(
-                totalOutstandingAmount,
+            int totalOutstandingAmountWithDiscount = totalOutstandingAmount + Math.Abs(appliedDiscountAmount);
```

Comment:
Instead of hiding this calculation within the `CalculateAdjustedSpendAmountCompleteOrder();` method (where it was simply being returned), I moved it here.  As you can see, `totalOutstandingAmountWithDiscount` is needed to calculate the correct `adjustedTaxAmount`. @